src/Sulu/Bundle/CoreBundle/Cache/StructureWarmer.php

```diff
@@ -30,15 +30,27 @@ public function __construct(StructureManagerInterface $structureManager)
         $this->structureManager = $structureManager;
     }
 
+    /**
+     * Warm up the cache for page and snippets.
+     *
+     * {@inheritdoc}
+     */
     public function warmUp($cacheDir)
     {
         // warmup the pages
         $this->structureManager->getStructures(Structure::TYPE_PAGE);
 
         // warm up the snippets
         $this->structureManager->getStructures(Structure::TYPE_SNIPPET);
+
+        return [];
     }
 
+    /**
+     * Return true for isOptonal method.
```

Comment:
Personally I don't like comments like these, would say they are superfluous and violate the DRY principle. I can easily spot the the `isOptional` function returns true in that case :wink:

Reply:
Hi, thx for your Feedback. The PHP fixer want a description here. So we need to add something. Please give me the correct sense and I will add it ;)

Reply:
Even if `{@inheritdoc}` is used? Who complains about that comment in general? PHPStan? Doesn't PHPStan realize that this is the same function as in the interface?

Reply:
@rogoit Let's keep the conversation in here :+1: PHP CS Fixer was running already before, and there was no problem, also without this PHPDoc, so that can't be the reason.